lib/puma/plugin/tailwindcss.rb

```diff
@@ -0,0 +1,65 @@
+require "puma/plugin"
+
+Puma::Plugin.create do
+  attr_reader :puma_pid, :tailwind_pid, :log_writer
+
+  def start(launcher)
+    return unless Rails.env.development?
+
+    @log_writer = launcher.log_writer
+    @puma_pid = $$
+    @tailwind_pid = fork do
+      Thread.new { monitor_puma }
+      system(*Tailwindcss::Commands.watch_command)
+    end
+
+    launcher.events.on_stopped { stop_solid_queue }
```

Comment:
Ah, taken from Solid Queue. This needs to be changed

Reply:
Whoops 😅